app/support/reservations/relay_support.rb

```diff
@@ -3,6 +3,7 @@ module Reservations::RelaySupport
   def can_switch_instrument_on?(check_off = true)
     return false if canceled?
     return false unless product.relay # is relay controlled
+    return false if product.offline?
```

Comment:
I hesitated bringing that in since would specifically need the `#offline_via_schedule_share` method, still isolated in #696 as I type this.